coverage.gradle

```diff
@@ -80,21 +80,12 @@ task generateCoverageReport(type: JacocoReport) {
 
 task unitTestsWithCoverage(type: GradleBuild) {
     // the 7.1.3 android gradle plugin has a bug where enableTestCoverage generates invalid coverage
-<<<<<<< HEAD
     startParameter.projectProperties.coverage = [enableTestCoverage:  false]
     tasks = [':vector:testGplayBtchapWithoutvoipWithoutpinningDebugUnitTest', ':matrix-sdk-android:testDebugUnitTest']
```

Comment:
In the diff, Element only have `tasks = ['testDebugUnitTest']`, can we have the same? I would expect having a single task without module prefix here